frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx

```diff
@@ -386,7 +385,9 @@ function ColumnPickerHeader({
     <ColumnPickerHeaderContainer>
       <ColumnPickerHeaderTitleContainer onClick={onClick} aria-label={t`Back`}>
         <Icon name="chevronleft" size={18} />
-        <ColumnPickerHeaderTitle>{children}</ColumnPickerHeaderTitle>
+        <Text fz="lg" fw="bold" lh="normal" c="inherit">
```

Comment:
I thought color would already be `inherit` by default, does Mantine apply a base text color by default?

Reply:
Nope, mantine applies a default color and then we use `defaultProps` in `getTextOverrides` to further dig ourselves into that hole